app/furniture/marketplace/marketplace.rb

```diff
@@ -30,6 +30,46 @@ def delivery_fee_cents
     end
     monetize :delivery_fee_cents
 
+    def stripe_account_link(refresh_url:, return_url:)
+      account = if stripe_account.blank?
+        Stripe::Account.create({type: "standard"}, {
+          api_key: stripe_api_key
+        }).tap do |account|
+          update(stripe_account: account.id)
+        end
+      else
+        Stripe::Account.retrieve(stripe_account, {
+          api_key: stripe_api_key
+        })
+      end
+
+      account_link = if account.details_submitted?
```

Comment:
Am I reading this right, that the only difference between the two if/else branches is the `type`? If so, maybe we can shorten this to something like:
```ruby
account_link = Stripe::AccountLink.create(
  {
    account: stripe_account,
    refresh_url: refresh_url,
    return_url: return_url,
    type: account.details_submitted? ? "account_update" : "account_onboarding"
  },
  {
    api_key: stripe_api_key
  }
)
```

Reply:
Good catch!